### ISSUE TITLE
Why exclude LayerNorm.bias from weight decay when finetuning?

### ISSUE THREAD

**Issue by xiaoda99** [huggingface/transformers#4832]
# ❓ Questions & Help

## Details

https://github.com/huggingface/transformers/blob/master/src/transformers/trainer.py#L306
In the original BERT implementation and in earlier versions of this repo, both LayerNorm.weight and LayerNorm.bias are decayed.

**A link to original question on Stack Overflow**:

**Comment by Jxu-Thu** [comment]
I have same question

**Comment by yangp725** [comment]
Check this [discussion](https://forums.fast.ai/t/is-weight-decay-applied-to-the-bias-term/73212/6)
